pkg/console/operator/sync_v400.go

```diff
@@ -35,6 +37,20 @@ import (
 	servicesub "github.com/openshift/console-operator/pkg/console/subresource/service"
 )
 
+var (
+	// metric: console_url{url="https://<url>"} 1
```

Comment:
Curious, there is the `console_url` from ?

Reply:
`Namespace: console` will make this `console_url`.